index.html

```diff
@@ -1,45 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    
+
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>My trip</title>
     <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-    <h1>ANYWAYS</h1>
-    <H2>Road trip around Tanzania</H2>
-    <hr>
-    <table>
-        <tr>
-            <td>OCT 2-6 <br>mo-fr(4 days)</td>
-            <td>Dar es salaam</td>
-           
-        </tr>
-        <tr>
-            <td>Oct 7-8 <br>sa=su(2 days)</td>
-            <td>Morogoro</td>
-        </tr>
-        <tr>
-            <td>Oct 9 <br>mo(1 days)</td>
-            <td>Mbeya</td>
-        </tr>
-        <tr>
-            <td>Oct 10=12<br>tu=th(3 days)</td>
-            <td>Kigoma</td>
-        </tr>
-        <tr>
-            <td>Oct 13-nov 5 <br>fr-su(2 days)</td>
-            <td>Mwanza</td>
-        </tr>
-        <tr>
-            <td>Nov 6-20<br>mo-mu(2 days)</td>
-            <td>Arusha</td>
-        </tr>
-
-    </table>
-    <hr>
-    <p>total: 1.5 months</p>
+    <h1>ANYWAYS →</h1>
+    <main>
+        <H2>Road trip around Tanzania</H2>
+        <hr>
+        <table>
+            <tr>
+                <td>
+                    <span class="date">Oct 2-6</span><br>
+                    <span class="duration">mo-fr (4 days)</span>
+                </td>
+                <td class="destination">Dar es Salaam</td>

```

Comment:
The main type selector in CSS is by class. You can refer to these blocks in you CSS as `.date { ... }`.